### PR TITLE
Release v3.3.0-alpha.2

### PR DESCRIPTION
**Pull request by GretaD**
info.xml is already changed https://github.com/nextcloud/mail/commit/8041d8115347c22643750fee66f90877b8aa21cd

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +30,15 @@ All notable changes to this project will be documented in this file.
...
 - Overlapping of primary menu with recipient
+- Editing state on new message modal
+- URL-encode all parameters to preserve special characters
+- Make nested toolbar working for signature
+- Fix force sending emails
+- Make system address book searchable not just full matches
+- Add preview enhancement job to new accounts joblist
+- Clear cache before returning
+- Mailboxes not being synced due to short circuiting
+-  Handle attachments without transfer encoding properly
```
Comment (ChristophWurst) [inline CHANGELOG.md:41]:
```suggestion
- Handle attachments without transfer encoding properly
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +30,15 @@ All notable changes to this project will be documented in this file.
 - Allow dynamic autoloading for classes added during upgrade
 - Re-open account settings
 - Overlapping of primary menu with recipient
+- Editing state on new message modal
```
Comment (ChristophWurst) [inline CHANGELOG.md:33]: not a user facing change AFAIK

File: CHANGELOG.md [hunk 3]
```diff
@@ -26,6 +30,15 @@ All notable changes to this project will be documented in this file.
...
+- URL-encode all parameters to preserve special characters
+- Make nested toolbar working for signature
+- Fix force sending emails
+- Make system address book searchable not just full matches
+- Add preview enhancement job to new accounts joblist
+- Clear cache before returning
```
Comment (ChristophWurst) [inline CHANGELOG.md:39]:
```suggestion
- Provisioning config caching
```

**Review by ChristophWurst — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (GretaD) [reply CHANGELOG.md]: the _go back_ button didnt work because of the editing state still existing on the template, so I rephrased the sentence.

**Pull request merged**